tests/async/test_immutable_migration_runner.py

```
@@ -29,12 +29,15 @@ def release_lock(self): self.events.append("unlock")
     def require_baseline(self, manifest): self.events.append("baseline")
     def applied(self): return list(self.rows)
     def apply(self, migration):
+        """Trace an apply, or raise when the test asked this stage to fail."""
         self.events.append(f"apply:{migration.migration_id}")
         if self.fail_apply: raise runner.MigrationContractError("synthetic apply failure")
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)